config/src/main/java/org/springframework/security/config/annotation/web/configurers/HeadersConfigurer.java

```diff
@@ -31,8 +31,6 @@
 import org.springframework.security.web.header.HeaderWriterFilter;
 import org.springframework.security.web.header.writers.*;
 import org.springframework.security.web.header.writers.ReferrerPolicyHeaderWriter.ReferrerPolicy;
-import org.springframework.security.web.header.writers.XContentTypeOptionsHeaderWriter;
```

Comment:
I really appreciate this kind of cleanup as well, but let's keep it out of this PR, given the description.

Reply:
I'll have to see how this can be split up.

The conversion to lambdas made many imports redundant, so I ran the "unused imports" cleanup which also removed some imports which were already redundant before